src/main/java/thePackmaster/patches/contentcreatorpack/PreDrawPatch.java

```diff
@@ -0,0 +1,19 @@
+package thePackmaster.patches.contentcreatorpack;
+
+import com.evacipated.cardcrawl.modthespire.lib.SpirePatch;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+
+public class PreDrawPatch {
+    public static boolean DRAWN_DURING_TURN = false;
+
+    @SpirePatch(
+            clz = AbstractPlayer.class,
+            method = "applyStartOfTurnRelics"
+    )
+    public static class AbstractPlayerApplyStartOfTurnRelicsPatch {
+        public static void Prefix(AbstractPlayer __instance) {
```

Comment:
With how this is implemented, effects that draw additional cards at the start of your turn will count as draw for this card That's something I find unintuitive -- draw like that feels like part of your pre-turn draw to me, and that's where people have ended up with similar effects to this already in the Packmaster repo, such as the card Trade Beacon (see TradeBeaconPower for how it does things.

You can avoid this by targeting `atStartOfTurnPostDraw` instead, and having this addToBot an action that sets these flags (so that all the actions in the queue resolve, which will include additional draw at the start of the turn, and then these flags get set).

Reply:
I believe that's what gets done currently? The VoyagedPatch adds an action at post draw which then enables the effect. This Pre-Draw patch is what sets the flag to false before all the draw is incurred.